### PR TITLE
Create rx wrapper

### PR DESCRIPTION
**Pull request by estebanarrua**

### CODE REVIEW

File: Example/RxSectionedCollectionView/ViewControllers/SectionedCollectionViewController.swift [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+//  SectionedCollectionView
+//
+//  Created by Esteban Arrua on 6/7/18.
+//  Copyright © 2018 Hattrick. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import RxSwift
+import SectionedCollectionView
```
Comment (martinzp) [inline Example/RxSectionedCollectionView/ViewControllers/SectionedCollectionViewController.swift]: is this import needed? shouldn't it be included with the Rx version?

File: Example/RxSectionedCollectionView/ViewController.swift [hunk 2]
```diff
@@ -1,6 +1,6 @@
 //
 //  ViewController.swift
-//  RxSectionedCollectionView
+//  SectionedCollectionView
```
Comment (martinzp) [inline Example/RxSectionedCollectionView/ViewController.swift]: if this viewcontroller is not used, can we delete it?

File: Example/RxSectionedCollectionView/ViewControllers/SectionedCollectionViewController.swift (hunk 1 above)
Comment (estebanarrua) [reply Example/RxSectionedCollectionView/ViewControllers/SectionedCollectionViewController.swift]: Yes, it is needed. The Rx only include de rx function, to define and use the SectionedCollectionView function and clases it needed import the SectionedCollectionView. I search in RxAlamonfire and RxPermission, and they did the same.

Comment (martinzp) [reply Example/RxSectionedCollectionView/ViewControllers/SectionedCollectionViewController.swift]: cool

**Pull request merged**